.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: Test
+on:
+  pull_request:
+    paths:
+      - 'src/**'
+      - '.github/workflows/test.yml'
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - ready_for_review
+  workflow_dispatch:
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 'latest'
+      - run: npm install
```

Comment:
I think we want `npm ci` https://stackoverflow.com/questions/52499617/what-is-the-difference-between-npm-install-and-npm-ci

Reply:
```suggestion
      - run: npm ci
```